jobs/dcp1-version/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: dcp1-version
+description: A Helm chart for Kubernetes
```

Comment:
Where does this description appear? should it link to the readme?

Reply:
I don't think it's necessary. These fields are more for charts that are being published. Similiar to description in package.json